docs/atlascli/command/atlas-dbusers-certs-create.txt

```diff
@@ -14,6 +14,10 @@ atlas dbusers certs create
 
 Create a new Atlas-managed X.509 certificate for the specified database user.
 
+You can't use this command to create certificates if you are managing your own Certificate Authority (CA) in self-managed X.509 mode.
+		
+The user you specify must authenticate using X.509 certificates.
```

Comment:
I would put the sentence on line 19 before the one on line 17 and remove spaces between them.